data/nat/basic.lean

```diff
@@ -859,4 +859,18 @@ lemma with_bot.add_eq_one_iff : ∀ {n m : with_bot ℕ}, n + m = 1 ↔ (n = 0 
 | (some n) (some (m + 1)) := by erw [with_bot.coe_eq_coe, with_bot.coe_eq_coe, with_bot.coe_eq_coe,
     with_bot.coe_eq_coe, with_bot.coe_eq_coe]; simp [nat.add_succ, nat.succ_inj', nat.succ_ne_zero]
 
+-- induction
+
+lemma induction_from_start (P : nat → Prop) {m} (h0 : P m) (h1 : ∀ n ≥ m, P n → P (n + 1)) : ∀ n ≥ m, P n :=
+begin
```

Comment:
You could use `nat.less_than_or_equal.rec` for a significantly shorter proof:
```lean
begin
    apply nat.less_than_or_equal.rec h0,
    exact h1
end
```

Reply:
The given proof makes more sense when `P : nat -> Sort l` instead of `Prop`

Reply:
Indeed, this is much more efficient. Thanks!